company/values.md

```diff
@@ -1,15 +1,34 @@
 # Sourcegraph values
 
-Our values are:
+Our values reflect who we are and how we work.
 
-## People
+## People first
 
-Together we are advancing technology for the good of people all around the world. We will attract, hire and retain the best teammates in the world and treat everyone in a first-class manner.
+[Our amazing team](teammate_qualities.md) is what makes Sourcegraph special. Together we are advancing technology for the good of people all around the world and we treat everyone in a first-class manner.
 
-## Journey
+## Customer driven progress
 
-The collection of moments, experiences, and memories that the team shares as we make progress: the light moments, the laughter, the team coming together to solve a problem.
+We have a [vision](strategy.md#vision) to build a product that is loved and valued by every company and developer. We learn what our customers want by understanding the challenges that they face, iterating quickly, and soliciting feedback. We feel good when we help other people, we are passionate about developer tools, and our business succeeds when customers are happy to pay for the value that we provide.
 
-## Progress
+Being customer driven does not mean we build bespoke features. Doing so would be a very inefficient and costly way to achieve our vision. Instead, we use customer feedback as data to help us identify work that would be valuable to a broad set of customers, and then directly engage with specific customers to validate that work.
 
-The continuous march toward achieving our mission as a team: the milestones, the successes, and the breakthroughs.
\ No newline at end of file
+## Open and transparent
+
+We build Sourcegraph (both the company and the product) in an open and transparent way. This means many things are public and open for input and contributions:
+
+- Our [code](https://github.com/sourcegraph/sourcegraph)
+- Our [handbook](../handbook/index.md)
+- Our [purpose, mission, vision, strategy, and product direction](strategy.md)
+- Our [OKRs](okrs/index.md)
+
+Being open and transparent builds trust with our customers and within our team. It makes it easy for us to partner and collaborate with our customers so we make Sourcegraph the best that it can be. It enables our teammates to work autonomously to solve tough problems while maintaining alignment.
+
+Transparency is hard. It requires extra effort and excellent communication. It means it is possible to observe things that are inconsistent or incomplete. We think this is a good thing because it gives everyone visibility into the process, not just the outcome, and an opportunity to contribute to that process at any point in time.
+
+## Servant leadership
```

Comment:
I like how this works on multiple levels: people in leadership roles practice servant leadership in regards to their team and Sourcegraph itself, as a "vessel and playground", does it to everyone at Sourcegraph.